ChromProcess/Loading/peak/peak_from_file.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+
+
+def peak_rt_from_file(chromatogram, Peakfile):
+    '''
+    Get peak retention time indices from a PeakCollections file.
+    chromatogram: Chromatogram object
+        Chromatogram object to which the peaks should be added
+    Peakfile: string
+        Location of peak collection csv
+    
+    Returns
+    -------
+    peak_indices: ndarray  
+        array of peak indices
+    '''
+    import pandas as pd
+    df = pd.read_csv(Peakfile,index_col=False)
+    peak_retention_times = df.iloc[3:,0].values #This function is very specific to the file type
+    peaks_indices = np.empty(0,dtype='int64') #the functions written to find the start and end of the peak relies on indexes, so we need to convert the retention times to their corresponding indexes.
+    for p_rt in peak_retention_times:
```

Comment:
Does peak_indices_to_times() do all of this?

Reply:
Its the reverse, so I guess this could be outsourced to a times_to_peak_indices()